docs/root/configuration/http_filters/tap_filter.rst

```diff
@@ -0,0 +1,122 @@
+.. _config_http_filters_tap:
+
+Tap
+===
+
+* :ref:`v2 API reference <envoy_api_msg_config.filter.http.tap.v2alpha.Tap>`
+* This filter should be configured with the name *envoy.filters.http.tap*.
+
+.. attention::
+
+  The tap filter is experimental and is currently under active development. There is currently a
+  very limited set of match conditions, output configuration, output sinks, etc. Capabilities will
+  be expanded over time and the configuration structures are likely to change.
+
+The HTTP tap filter is used to tap HTTP traffic. At a high level, the configuration is composed of
+two pieces:
+
+1. Match configuration: a list of conditions under which the filter will match an HTTP request
+   and begin a tap session.
+2. Output configuration: a list of output sinks that the filter will write the matched and tapped
+   data to.
+
+Each of these concepts will be covered in the following example configuration section.
```

Comment:
Maybe point out that this is a running example; it actually spans multiple sections to cover the concepts introduced above.